src/cityreader/cityreader.py

```diff
@@ -15,14 +23,20 @@
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
 cities = []
+with open('cities.csv', 'r') as city_data:
+    reader = csv.reader(city_data)
+    next(reader, None)  # skip the headers
+    city_list = list(reader)
+
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+
     return cities
 
+
 cityreader(cities)
 
 # Print the list of cities (name, lat, lon), 1 record per line.
```

Comment:
you want to print city with name and lat and lon
you have an empty cities list,
maybe in your function you can append the info and print after,
so you can do something like for row in reader: cities.append(City(row[0],row[3],row[4]))

then print after you invoke cityreader

for c in cities:
print(c)